server/app/lib/fusor/validators/deployment_validator.rb

```diff
@@ -79,6 +80,16 @@ def validate(deployment)
           if !deployment.rhev_is_self_hosted && deployment.rhev_hypervisor_hosts.count < 1
             deployment.errors[:rhev_hypervisor_hosts] << _('RHEV deployments must have at least one Hypervisor')
           end
+
+          if deployment.rhev_engine_host.name.gsub(/[^0-9A-Za-z\-\.\_]/, '*') != deployment.rhev_engine_host.name
```

Comment:
This logic where you're substituting characters seems a bit convoluted. Why not just directly check if the host name contains `0-9A-Za-z\-\.\_`?